shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/remote/body/LogInRequestBody.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.avishek.phong.kmmic.data.remote.body
+
+import co.nimblehq.avishek.phong.kmmic.BuildKonfig
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+private const val GRANT_TYPE_PASSWORD = "password"
+
+@Serializable
+data class LogInRequestBody(
```

Comment:
Please separate the Backend task to another PR next time, this PR is quite large and a bit hard to review now 🙏

Reply:
I'll do that next time